tests/vcr/tests.test_subscribe/Test_SubscriptionRegistry.test_register_unregister.yaml

```diff
@@ -39,4 +39,44 @@ interactions:
     status:
       code: 200
       message: OK
+- request:
```

Comment:
Empty body and empty response body, is that right?

Reply:
That is correct. These requests use only the headers to pass values.

More details can be found in the [UPnP Device Architecture](http://www.upnp.org/specs/arch/UPnP-arch-DeviceArchitecture-v1.0-20080424.pdf) spec starting around page 64.